Resources/views/CRUD/base_edit.html.twig

```diff
@@ -12,7 +12,7 @@ file that was distributed with this source code.
 {% extends base_template %}
 
 {% block title %}
-    {% if admin.id(object) is not null %}
+    {% if objectId is not null %}
```

Comment:
This is a BC-break for people that extend the `CrudController`. A solution could be this:

`{% if objectId|default(admin.id(object)) is not null %}`